keda/templates/metrics-server/deployment.yaml

```diff
@@ -113,6 +113,7 @@ spec:
           - --port={{ .Values.prometheus.metricServer.port }}
           - --secure-port={{ .Values.service.portHttpsTarget }}
           - --logtostderr=true
+          - --stderrthreshold=ERROR
```

Comment:
Could you please add a parameter for this? So users can specify the threshold?

Reply:
sure @zroubalik